src/javascripts/components/forms/updateSector.js

```diff
@@ -0,0 +1,59 @@
+import systemsData from '../../helpers/data/systemsData';
+import sectorData from '../../helpers/data/sectorData';
+
+const updateSectorForm = (sectorObject) => {
+  $('#update-sector-form').html(`
+      <h2>Update sector</h2>
+      <div id="success-message"></div>
+      <form>
+        <div id="error-message"></div>
+        <div class="form-group">
+          <label for="name">Name</label>
+          <input type="text" class="form-control" id="name" value="${sectorObject.name}">
+        </div>
+        <div class="form-group">
+          <label for="systems">System</label>
```

Comment:
Again, why would you ask for system if you wouldn't display it somewhere on the sector card. Just curious!

Reply:
This is part of our stretch goal. Just laying the groundwork here. We can take it out if needed.

Reply:
No need if there is a plan, go for it!